examples/structured-keys/structured_keys.description

```diff
@@ -1,3 +1,3 @@
-// Key fields can be structured where they belong to any subtype of plain data. The Value of key field
-// must be immutable. Initializer of `readonly` field will be constructed as immutable. In other cases,
+// Key fields can be structured in places where they belong to any subtype of plain data. The value of the key field
```

Comment:
What does "Key fields can be structured in places" mean? 🤔  From how it was originally phrased, I understood it to mean that key fields can be of structured types which are subtypes of `anydata`. I'm didn't quite get the new phrasing 🤔

Reply:
Oh the original phrase `Key fields can be structured where they belong to any subtype of plain data.` in a literal sense read to me somewhat like:

> key fields can be structured types wherever, as long they belong to any subtype of plain data.

 That's why I added the `in places ` part for it to read better.

Reply:
I see. @MaryamZi thoughts? 🤔

Reply:
How about

```suggestion
// Key fields can be structured if they belong to any subtype of plain data. The value of the key field
```